src/Curl/Curl.php

```diff
@@ -765,7 +754,57 @@ public function setOpt($option, $value)
         }
 
         $this->options[$option] = $value;
-        return curl_setopt($this->curl, $option, $value);
+        return (bool) curl_setopt($this->curl, $option, $value);
```

Comment:
(sometimes) I try to force ably cast return values if I can.  It's, um, a weird habit.  Follows along with my too many spaces in docblocks.